phpBB/config/default/routing/storage.yml

```diff
@@ -0,0 +1,22 @@
+phpbb_storage_avatar_controller:
```

Comment:
I was looking at this:
https://github.com/phpbb/phpbb/blob/master/phpBB/config/default/routing/report.yml